src/assets/scss/components/_alert.scss

```diff
@@ -0,0 +1,78 @@
+.shd-alert {
+  &--info {
+    border-radius: 0.25rem;
+    border: 0.063rem solid $blue-100;
```

Comment:
the only thing is different is border-color in all variants why define the border-radius and other border properties multiple times.